web_timeline/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Web timeline",
     "summary": "Interactive visualization chart to show events in time",
-    "version": "16.0.1.1.5",
+    "version": "16.0.1.1.6",
```

Comment:
Please don't change manually the module version.

Reply:
My bad. Just removed it :)